docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "2"
```

Comment:
We use 2.1 for `devstack`, should this be upgraded to match it? Though I think compose is up to 3 now.

Reply:
If we need something from 3 I am ok to upgrade to it in the future.  I will set it to 2.1 for now if that will work.